eve/main.yml

```diff
@@ -123,6 +135,8 @@ stages:
               bash -c '
               source metalk8s-%(prop:product_version_prev)s/VERSION &&
               echo %(prop:product_version_prev)s.$VERSION_PATCH$VERSION_SUFFIX'
+
+      - ShellCommand: *download_pdf_doc
```

Comment:
So now we'd end up with a 'previous version' build/ISO containing the docs of the 'current version'?

Reply:
Yeah, I guess for upgrade/downgrade we don't care about the embedded doc anyway.
It's just a matter of having a build that works.

Now if we have a simple way to retrieve the matching doc, why not.
If it's more involved then I'm not sure it's worth the effort.

Reply:
Can't we simply not embed any doc in the resulting ISO, then?

Reply:
But we always want the doc in the ISO, no?
Why should me make it optional? Only to workaround a quirk in our CI workflow?

This ISO is only built for testing upgrade/downgrade, it won't be uploaded anywhere to be reused.
I would say that embdedding a wrong PDF in a transient ISO is a lesser evil than making doc embedding optional in the buildchain (which could bite us later it the codepath is triggered by error when we build the real ISO we ship)